### ISSUE TITLE
Endpoints aren't being applied fully

### ISSUE THREAD

**Issue by tomvachon** [rebuy-de/aws-nuke#154]
I'm finding place where the operations are not supported in said region or the API endpoint doesnt exist.  Maybe its not merged or I missed some guidance, but I though the endpoint discovery in the core code was fixing this

**Comment by svenwltr** [comment]
Can you give an example resource and region?

**Comment by tomvachon** [comment]
```
ERRO[0025] Listing SNSPlatformApplication failed. Please report this to https://github.com/rebuy-de/aws-nuke/issues/new.
    !!! InvalidAction: Operation (ListPlatformApplications) is not supported in this region
    !!! 	status code: 400, request id: 9326e2f0-6aa6-5641-9cb5-b1dc325df7ce
```

```
ERRO[0047] Listing MQBroker failed. Please report this to https://github.com/rebuy-de/aws-nuke/issues/new.
    !!! RequestError: send request failed
    !!! caused by: Get https://mq.us-west-1.amazonaws.com/v1/brokers?maxResults=100: dial tcp: lookup mq.us-west-1.amazonaws.com: no such host
```

**Comment by svenwltr** [comment]
Ok, this is because MQ does not have an endpoint entry at all: https://github.com/rebuy-de/aws-nuke/blob/ba4e46cbe0195860119092babbae872d56bfe471/pkg/awsutil/session.go#L121-L125

I don't know how we should handle this case.

**Comment by tomvachon** [comment]
@svenwltr how about this?

1. If we get back a nil return value branch to this logic
2. Test DNS to see if the endpoint exists

**Comment by svenwltr** [comment]
@rebuy-de/prp-aws-nuke Your opinions on this approach?

**Comment by bjoernhaeuser** [comment]
Feels wrong to me to hack this together, imho a SDK thingy. But if they do not deliver that, the proposed solution sounds good to me.

**Comment by tomvachon** [comment]
Also, if its not in the list (e.g MediaStoreData), we should presume its regional, not global.  Right now the code thinks its "both"

**Comment by tomvachon** [comment]
I agree @bjoernhaeuser this shouldnt ever be "hit" but its cleaner than how it is now. Maybe we could throw up a "Warning" when we hit this instead of silently passing it by?  Something like:

"WARN: Unable determine automatically endpoint for $SERVICE.  DNS lookup failed for $SERVICE in $REGION --- SKIPPING SERVICE!"

**Comment by svenwltr** [comment]
Yes a warning is very important, because it could also appear on network issues.

**Comment by svenwltr** [comment]
This problem should be fixed in [v2.1.0-beta.1](https://github.com/rebuy-de/aws-nuke/releases/tag/v2.1.0-beta.1).
